packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -207,6 +209,15 @@ class AutomationRunner<TStep extends AutomationTriggerStepId> {
 
     return response
   }
+
+  async trigger(
```

Comment:
should it be made more obvious this triggers the automation rather than adding a trigger to the automation you're building?